Cargo.toml

```diff
@@ -24,7 +24,7 @@ markov-chain = { git = "https://github.com/jelni/markov-chain" }
 md5 = "0.7"
 oneshot = "0.1"
 rand = "0.9"
-reqwest = { version = "0.12", features = ["json", "stream"] }
+reqwest = { version = "0.12", default-features = false, features = ["json", "stream", "rustls-tls"] }
```

Comment:
is there a specific reason for using Rustls? i get it's _rewrite-it-in-rust_, but i haven't faced issues with native OS implementations yet, which are still faster

Reply:
With this change you can remove OpenSSL as a dependency in docker. You can revert if you want, I recommend rustls because it just works

Reply:
TDLib requires OpenSSL as a build and runtime dependency

> it just works

it's not like OpenSSL doesn't

Reply:
Getting rid of system libraries makes it easier to cross-compile and to create portable/static builds 👍